MapboxNavigation/Constants.swift

```diff
@@ -20,6 +20,10 @@ public let RouteControllerShouldReroute = Notification.Name(MBRouteControllerSho
 */
 public var RouteControllerMaximumDistanceBeforeRecalculating: CLLocationDistance = 50
 
+/*
+ Accepted deviation excluding horizontal accuracy before the user is considered to be off route.
+ */
+public var RouteControllerHorizontalAccuracyAnomaly: CLLocationDistance = 10
```

Comment:
This global variable doesn’t represent an anomaly; it represents the maximum deviation distance that can be considered an anomaly. How about calling this global `RouteControllerUserLocationSnappingDistance`, to describe the desired output instead of one specific input (which may not be the only output in the future)?